Formula/o/objc-run.rb

```diff
@@ -11,6 +11,12 @@ class ObjcRun < Formula
     sha256 cellar: :any_skip_relocation, all: "951d50ad3ee4ebb9d0717b4df365870b44626195378b9d5c64bcf7b320e8cc14"
   end
 
+  # no new commits since 2014-02-10
+  deprecate! date: "2022-01-01", because: :unmaintained
```

Comment:
I don't think this is a good reason to deprecate. Is it actually broken? It seems to really support macOS only since it uses `-framework` flags.

Reply:
okay, I can update the deprecate reason per this issue, https://github.com/iljaiwas/objc-run/issues/19

also [we have deprecated cocoapods](https://github.com/Homebrew/homebrew-cask/pull/105657) as well (seems referenced in that issue)

Reply:
I'm confused -- `cocoapods` is not deprecated on `master`? 

https://github.com/Homebrew/homebrew-core/blob/279dcc4faf5f2f62cf64a6ae8732974672b29249/Formula/c/cocoapods.rb

I'd rather we ask upstream for the status of the repository first before trying to deprecate it.

Reply:
fair enough, I will just remove the deprecation note then.